tests/repository_data/generate.py

```diff
@@ -119,12 +119,12 @@
 # about the target (i.e., file permissions in octal format.)
 octal_file_permissions = oct(os.stat(target1_filepath).st_mode)[4:]
 file_permissions = {'file_permissions': octal_file_permissions}
-repository.targets.add_target(target1_filepath, file_permissions)
-repository.targets.add_target(target2_filepath)
+repository.targets.add_target(os.path.basename(target1_filepath), file_permissions)
+repository.targets.add_target(os.path.basename(target2_filepath))
 
```

Comment:
As @lukpueh's hint in #933 was to make changes like this in order to re-generate the test repository data I wondered why we aren't making this change to the `generate.py` script, rather than hacking it each time?

I will dig into this and try to better understand it.